NOTES.rst

```diff
@@ -45,6 +38,13 @@ receive optical power values.
 The implementation was contributed by Pär Stolpe of Linköping University, and
 has been specifically tested on Alcatel Lucent Enterprise OmniSwitch AOS 8.
 
+psuwatch ipdevpoll plugin replaces powersupplywatch program
```

Comment:
Possibly a big enough change that it should be in 4.10/5.

Reply:
Which is why this is targeted to the master branch, not a stable branch - and the milestone set to 4.10 :-)